apps/site/components/Icons/InstallationMethod/N.tsx

```diff
@@ -0,0 +1,34 @@
+import type { FC, SVGProps } from 'react';
+
+const N: FC<SVGProps<SVGSVGElement>> = props => (
```

Comment:
is it official one or an invented one for this site ?

Reply:
Invented. (By the current `n` maintainer, if that helps.)

Reply:
okay go it